.gitignore

```diff
@@ -10,6 +10,7 @@ scripts/memory/project/src/pages/
 package-lock.json
 .turbo/
 .eslintcache
+.pnpm-store
```

Comment:
Is there a reason why this is needed in node 16 and above only?

Reply:
![image](https://user-images.githubusercontent.com/27261522/199173030-f0a7d261-3b97-4dc5-91fb-b7441d851b44.png)
My node version is 17, when I ran "pnpm install", this project generate this folder

Reply:
I didn't use Gitpob,
this file was generated when I ran Pnpm install with node 17

Reply:
I see. Looks like we also have a Docker config file too, so maybe this is fine. Thanks for clarifying!